modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/services/HealthCheckConsensus.scala

```diff
@@ -91,37 +93,86 @@ abstract class HealthCheckConsensus[
 
   def startOwnRound(key: HealthCheckKey) =
     clusterStorage.getPeers.flatMap { peers =>
-      createRoundId.flatMap {
+      createRoundId.map(HealthCheckRoundId(_, selfId)).flatMap {
         startRound(key, peers, _, OwnRound)
       }
     }
 
-  def participateInRound(key: HealthCheckKey, roundId: RoundId): F[Unit] =
+  def participateInRound(key: HealthCheckKey, roundId: HealthCheckRoundId): F[Unit] =
     clusterStorage.getPeers.flatMap { peers =>
       startRound(key, peers, roundId, PeerRound)
     }
 
   def startRound(
     key: HealthCheckKey,
     peers: Set[Peer],
-    roundId: RoundId,
+    roundId: HealthCheckRoundId,
     roundType: HealthCheckRoundType
-  ): F[Unit] = ???
+  ): F[Unit] =
+    Clock[F].monotonic
```

Comment:
1. If you need an actual time of the start of the round then `Clock[F].realTime` should be used instead of `monotonic`
2. It looks like you're not using the `startedAt` paramter at all.

Reply:
I'm going to pass it to the round and most likely I need monotonic instead of realTime. If so, I'll change that in next PR with actual usage in a particular round

Reply:
Ok, it's difficult for reviewers to tell the difference between what's not used yet and will be used later on and what's not used because the author forgot to remove it.